src/main/resources/META-INF/mods.toml

```diff
@@ -41,7 +41,7 @@ MineColonies is a colony simulator within Minecraft! There are numerous types of
     # Does this dependency have to exist - if not, ordering below must be specified
     mandatory=true #mandatory
     # The version range of the dependency
-    versionRange="[34.1.0,)" #mandatory
+    versionRange="[36.1.0,)" #mandatory
```

Comment:
The build.properties refers to 36.1.24; wasn't sure if I should use that exact version as a minimum or if it should be more lenient.  This seemed like an appropriate compromise.